src/add/observable/combineLatest.ts

```diff
@@ -1,6 +1,4 @@
 import {Observable} from '../../Observable';
 import {combineLatestStatic} from '../../operator/combineLatest';
 
-Observable.combineLatest = combineLatestStatic;
-
-export var _void: void;
```

Comment:
I'm not sure what you mean by that. This file is still considered an external module after removal of `export var _void` if that's what you mean, because it has top level `import`s.